### DISCUSSION TITLE
guide-R/analyses_avancees/classification-ascendante-hierarchique

### DISCUSSION THREAD

**Discussion by giscus** [larmarange/guide-R#57] [Forum]
# guide-R/analyses_avancees/classification-ascendante-hierarchique

https://larmarange.github.io/guide-R/analyses_avancees/classification-ascendante-hierarchique.html

**Comment by mascarpon** [comment]
Bonjour, 

Merci pour ces explications très détaillées. 

Une question concernant le choix du nombre d'axes d'une ACM à retenir pour la CAH : est-il recommandé d'utiliser les taux d'inertie modifiés de Benzecri ou de Greenacre pour déterminer le nombre d'axes expliquant une part satisfaisante de l'inertie ? 

Le taux d'inertie non modifié (celui utilisé dans votre texte) pouvant être considéré comme "pessimiste" dans le cas d'une ACM, l'utiliser ne conduit il pas à retenir un nombre inutilement grand d'axes ? 

Merci d'avance pour vos éclairages !

**Reply by larmarange** [reply]
Pour réaliser une CAH, vous pouvez décider de garder tous les axes. Cela n'est pas problématique. Il faudrait d'ailleurs vérifier si cela change beaucoup les résultats (tous les axes vs les principaux axes). Ne garder que les principaux axes c'est faire en sorte que ce qui est capturé dans les derniers axes jouent moins dans le calcul de la CAH.

Il me semble qu'il y a plusieurs écoles. Pour ma part, je base surtout sur les classes finales obtenues. Font-elles sens d'un point sociologique ? Arrivent-elles à capturer ce que je cherche à étudier ?

Concernant les taux d'inertie modifiés de Benzecri ou de Greenacre, je ne saurais dire. Le mieux serait peut être de poser la question aux développeurs des package FactoMineR et/ou ade4, sachant qu'ils sont francophones.

Cordialement
